test/e2e/tests/test_new_crd_auth.py

```
@@ -0,0 +1,142 @@
+
+# ─── Preflight: skip if model not deployed ──────────────────────────────────
+
+def _model_is_routable():
+    """Check that the ExternalModel exists and has an HTTPRoute."""
+    try:
+        model = _get_cr("externalmodel.inference.opendatahub.io", NEW_CRD_MODEL, NEW_CRD_NAMESPACE)
+        if model is None:
+            return False
+        route = _get_cr("httproute", NEW_CRD_MODEL, NEW_CRD_NAMESPACE)
+        return route is not None
+    except Exception:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)